Formula/mplayer.rb

```diff
@@ -20,6 +20,8 @@ class Mplayer < Formula
   end
 
   depends_on "yasm" => :build
+  depends_on "freetype" => :build
+  depends_on "libpng" => :build # required by freetype
```

Comment:
The CI is failing because `libpng` isn't a pure build-time dependency. I'd be surprised if `freetype` was as well, to be honest, but that's mildly more plausible.